spec/cypress/integration/nominations/NominateInductionCoordinator.feature

```diff
@@ -56,7 +56,7 @@ Feature: Nominate induction tutor
     And the page should be accessible
     And percy should be sent snapshot called "Start nominations name different"
 
-    Given user was created as "teacher" with email "different-user-type@example.com"
+    Given user was created with email "different-user-type@example.com"
```

Comment:
Out of curiosity, did this start to fail here, or is it a cheeky fix of an incorrect test?

Reply:
This started to fail here. This line created a teacher profile for the user, then the next line creates another